internal/store/leaf/recipients.go

```diff
@@ -71,7 +71,7 @@ func (s *Store) AddRecipient(ctx context.Context, id string) error {
 	debug.Log("new recipient: %q - existing: %+v", id, rs)
 
 	for _, k := range rs {
-		if k == id {
```

Comment:
This has the side effect of adding the same key ID to the recipients list twice. I don't like that.

What about moving this loop and the append to a small helper function that only returns an error when force is not set but only appends if the key is indeed missing? That way we'd still be able to re-encrypt although the recipients didn't change.

Or we just drop the error and always re-encrypt, even w/o the force flag?
That would make the implementation more lightweight and easier to use. At the expense that users might accidentially re-encrypt the store (I don't think that's a problem.)

Reply:
So when gopass does "Starting reencrypt" and you ctrl+c out of it, it leaves git in a weird state. Since users might accidentally select the wrong option and try to ctrl+c out of it, I dont think we should just automatically re-encrypt when the same key is selected. Ive implemented a confirm prompt in my latest commit that explains why someone might want to re-encrypt their entire directory. Let me know what you think.